docs/TuxActions.md

```diff
@@ -330,12 +338,19 @@ Taking everything we have learned let's look at a more realistic example of usin
       }
     });
 
+    var messageActions = Actions.createActionCategory({
+      category: 'messages',
+      source: 'message components',
+      actions: ['get', 'create', 'delete']
+    });
+
     messageActions.register(messageStore, {
       get: messageStore.onGet,
       create: messageStore.onCreate,
       delete: messageStore.onDelete
     });
 
+    //the messageActions category wraps all of the business logic associated with messages, fully encapsulating all of the work associated with an app action
     messageActions.before('get', function (nextCallback, actionBody) {
       ajaxRequest('/messages', 'GET').then(function (results) {
         nextCallback(results); //dispatch results of Ajax
```

Comment:
Recommend adding a summary of what this example is doing and maybe how it takes advantage of `Actions`.